Protocol.md

```diff
@@ -949,11 +949,14 @@ When a PAYLOAD is fragmented, the Metadata MUST be transmitted completely before
 
 For example, a single PAYLOAD with 20MB of Metadata and 25MB of Data that is fragmented into 3 frames:
 
+##### Payload with `next` flag set
```

Comment:
I don't think this needs to be a sub-section with a heading, especially coming after a `":"`. Rather it can be just the first example you see. So basically I suggest removing the sub-heading completely. Then after that, below for the second example, simply add a sentence before it. I'll add a concrete suggestion there.